bbl_screen-patch/patches/printerui/qml/settings/ConsolePage.qml

```diff
@@ -270,7 +355,19 @@ Item {
             }
         }
     }
+    
 
+    function navigateHistory(up) {
+        if (cmdHistory.length === 0) return;
+
+        if (up) {
+            historyIndex = (historyIndex + 1) % cmdHistory.length;
+        } else {
+            historyIndex = (historyIndex - 1 + cmdHistory.length) % cmdHistory.length;
+        }
+
+        inputText = cmdHistory[historyIndex];
+    }
```

Comment:
This is an old feature! See https://github.com/X1Plus/X1Plus/blob/d5f2386fb766b95dfb90b24ac5a7991027d29135/bbl_screen-patch/patches/printerui/qml/settings/ConsolePage.qml#L377

You are right that the conditional is unnecessary. I will modify this

Reply:
> Sigh. I really wish this new feature were not buried in an already large PR. I really really really want PRs to be one thing at a time so that they are more reviewable. We have had history-related bugs before, so this actually needs to be looked at... and it's really hard to do that when there are an enormous amount of other diffs to the same file.
> 
> In this specific case, if `up` is always `true`, the conditional should be removed -- code that is never exercised (and, therefore, never tested!) should not be added.

Why don't I pull this into a different PR?